core/src/main/resources/application.conf

```diff
@@ -0,0 +1,106 @@
+collector {
+  enableDefaultRedirect = false
+
+  paths {
+
+  }
+
+  crossDomain {
+    enabled = false
+    domains = [ "*" ]
+    secure = true
+  }
+
+  cookieBounce {
+    enabled = false
+    name = "n3pc"
+    fallbackNetworkUserId = "00000000-0000-4000-A000-000000000000"
+  }
+
+  cookie {
+    enabled = true
+    expiration = 365 days
+    name = sp
+    domains = [
+
+    ]
+    fallbackDomain = "acme1.com"
+    secure = true
+    httpOnly = false
+    sameSite = "None"
+  }
+
+  doNotTrackCookie {
+    enabled = false
+    name = ""
+    value = ""
+  }
+
+  ssl {
+    enable = false
+    redirect = false
+    port = 9543
+  }
+
+  p3p {
+    policyRef = "/w3c/p3p.xml"
+    CP = "NOI DSP COR NID PSA OUR IND COM NAV STA"
+  }
+
+  prometheusMetrics {
+    enabled = false
+  }
+
+  rootResponse {
+    enabled = false
+    statusCode = 302
+    headers = {}
+    body = "302, redirecting"
```

Comment:
Strictly speaking should be [empty string for backwards compatibility](https://github.com/snowplow/stream-collector/blob/2.3.1/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala#L92).  On the other hand, I'm fairly sure nobody ever sees this string.